source/_components/sensor.coinmarketcap.markdown

```diff
@@ -14,17 +14,19 @@ ha_iot_class: "Cloud Polling"
 ---
 
 
-The `coinmarketcap` sensor platform displays various details about a crypto currency provided by [CoinMarketCap](http://coinmarketcap.com/).
+The `coinmarketcap` sensor platform displays various details about a cryptocurrency provided by [CoinMarketCap](http://coinmarketcap.com/).
 
 To add the CoinMarketCap sensor to your installation, add the following lines to your `configuration.yaml`:
 
 ```yaml
 # Example configuration.yaml entry
 sensor:
   - platform: coinmarketcap
+    currency: ethereum
+    display_currency: EUR
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.